tests/arm/Tests.lst

```diff
@@ -4681,7 +4681,7 @@ RelativePath=GC\Regressions\dev10bugs\536168\536168\536168.cmd
 WorkingDir=GC\Regressions\dev10bugs\536168\536168
 Expected=0
 MaxAllowedDurationSeconds=600
-Categories=EXPECTED_PASS;Pri1
+Categories=EXPECTED_FAIL;Pri1
```

Comment:
Is there a bug tag to add?

Reply:
Guess I could tag it with https://github.com/dotnet/coreclr/issues/3392, like issues.targets. Not sure how much value there is in doing so.

Looks like arm64 says:
```
Categories=EXPECTED_FAIL;11534;LONG_RUNNING;NEW;Pri1
```

but 11534 is arm64-specific.

fyi, https://github.com/dotnet/coreclr/issues/11534 looks very out of date...